### PR TITLE
parted: refresh

### PR DESCRIPTION
**Pull request by bl-ue**

### CODE REVIEW

File: pages/linux/parted.md [hunk 1]
```diff
@@ -7,30 +7,30 @@
...
 
 - Select a disk in interactive mode:
 
 `select {{/dev/sdX}}`
 
-- Interactively create a 16GB partition with a given filesystem:
+- Create a 16GB partition with the specified filesystem in interactive mode:
 
-`mkpart {{primary|logical|extended}} {{filesystem}} {{0%}} {{16G}}`
+`mkpart {{primary|logical|extended}} "{{btrfs|ext2|ext3|ext4|fat16|fat32|hfs|hfs+|linux-swap|ntfs|reiserfs|udf|xfs}}" {{0%}} {{16G}}`
```
Comment (navarroaxel) [inline pages/linux/parted.md:28]: `hfs+` needs `"`? I think not.

Comment (bl-ue) [reply pages/linux/parted.md:28]:
```suggestion
`mkpart {{primary|logical|extended}} {{btrfs|ext2|ext3|ext4|fat16|fat32|hfs|hfs+|linux-swap|ntfs|reiserfs|udf|xfs}} {{0%}} {{16G}}`
```

**Review by navarroaxel — APPROVED** [review]

**Review by marchersimon — APPROVED** [review]

**Comment by marchersimon** [comment]
Should we generally use `a specified` or `the specified`? Because I see both versions quite often and it would be cool if we could normalize that.

**Comment by bl-ue** [comment]
I prefer "the specified," but I think it's even better to simplify it (and the surrounding context if necessary) to "a," e.g.:
```diff
-Copy the specified file to the specified directory:
+Copy a file to a directory: 
```
I agree with your: normalize, normalize, normalize, normalize everything you can!  It's like a batch, self-paced review of all of the PRs 👍🏻

**Review by einverne — APPROVED** [review]
Nice work!

**Pull request merged**